src/components/BusinessForm.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState, ChangeEvent } from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import { BusinessQuery } from '../models/Business';
+
+interface BusinessFormProps {
```

Comment:
A common naming convention for interfaces is to prefix the name with a capital i 
```suggestion
interface IBusinessFormProps {
```

Reply:
Apparently TypeScript is one of the languages where this convention doesn't apply https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface

Reply:
That's a terrible reason... vs at-a-glance association which is why we also do the capitalization conventions